src/test/java/core/basesyntax/service/strategy/TransactionProcessorStrategyTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.service.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.TransactionProcessor;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TransactionProcessorStrategyTest {
+    private static List<FruitTransaction> transactions;
+    private static Map<String, Integer> fruits;
+    private static TransactionProcessor strategy;
+
+    @BeforeClass
+    public static void setUp() {
+        transactions = new ArrayList<>();
+        fruits = new HashMap<>();
+        strategy = new TransactionProcessorStrategy();
+    }
+
+    @After
+    public void tearDown() {
+        transactions.clear();
+        fruits.clear();
+    }
+
+    @Test
+    public void process_InitialFruits_ok() {
+        transactions.add(new FruitTransaction(Operation.BALANCE, "banana", 15));
+        strategy.process(transactions, fruits);
+        int expected = 15;
```

Comment:
```suggestion
        FruitTransaction fruitTransaction  = new FruitTransaction(Operation.BALANCE, "banana", 15);
        transactions.add();
        strategy.process(transactions, fruits);
        int expected = fruitTransaction.getQuantity();
```
let's avoid hardcode where it's possible, check every test method where you have similar case and try to avoid hardcode